cli/connhelper/connhelper.go

```diff
@@ -53,3 +56,19 @@ func GetCommandConnectionHelper(cmd string, flags ...string) (*ConnectionHelper,
 		Host: "http://docker",
 	}, nil
 }
+
+func multiplexingArgs() []string {
+	if v := os.Getenv("DOCKER_SSH_NO_MUX"); v != "" {
```

Comment:
can you document the new environment variables in https://github.com/docker/cli/blob/master/docs/reference/commandline/cli.md#environment-variables ?